pkg/sloop/store/typed/relationshiptable.go

```diff
@@ -0,0 +1,72 @@
+package typed
+
+import (
+	"fmt"
+	"github.com/salesforce/sloop/pkg/sloop/common"
+	"github.com/salesforce/sloop/pkg/sloop/store/untyped"
+	"time"
+)
+// Relationship table to keep node/rs to pods mapping
+
+// Key is /<partition>/<kind>/<namespace>/<name>/<uid>
+//
+// Partition is UnixSeconds rounded down to partition duration
+// Kind is kubernetes kind, starts with upper case
+// Namespace is kubernetes namespace, all lower
+// Name is kubernetes name, all lower
+// Uid is kubernetes $.metadata.uid
+
+type RelationshipKey struct {
+	PartitionId string
+	Kind        string
+	Namespace   string
+	Name        string
+	Uid         string
+}
+
+func (*RelationshipKey) TableName() string {
+	return "relation"
```

Comment:
consider moving this to a named constant
```
const (
   relationshipTableName = "relation"
)
```

Reply:
updated